<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientAppointment</class>
 <widget class="QWidget" name="PatientAppointment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>End</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QTimeEdit" name="startTime"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4" colspan="2">
    <widget class="QTimeEdit" name="endTime"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>DoctorName</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>DoctorId</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="5">
    <widget class="QDateEdit" name="date"/>
   </item>
   <item row="0" column="1" colspan="5">
    <widget class="QLineEdit" name="doctorName"/>
   </item>
   <item row="4" column="0" colspan="6">
    <widget class="QSlider" name="timeSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="5">
    <widget class="QLineEdit" name="doctorId"/>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QPushButton" name="closeWindow">
     <property name="text">
      <string>Close Window</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3" colspan="3">
    <widget class="QPushButton" name="submit">
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
